onmt/inputters/text_dataset.py

```diff
@@ -72,12 +65,8 @@ def __init__(self, fields, src_examples_iter, tgt_examples_iter,
         super(TextDataset, self).__init__(examples, fields, filter_pred)
 
     @staticmethod
-    def collapse_copy_scores(scores,
-                             batch,
-                             tgt_vocab,
-                             src_vocabs,
-                             batch_dim=1,
-                             batch_offset=None):
+    def collapse_copy_scores(scores, batch, tgt_vocab, src_vocabs,
+                             batch_dim=1, batch_offset=None):
```

Comment:
While making Datasets cleaner, we need to take out this method from here.
Maybe later.